gen/gen.go

```diff
@@ -336,31 +433,29 @@ func (g *Generator) FunctionReturns(f Function) string {
 func (g *Generator) FunctionArgs(f Function, o *Object) string {
 	sb := &strings.Builder{}
 	if o != nil {
-		typeSuffix := g.TypeSuffixForNamespace(o.Namespace)
 		if len(f.Args) > 0 {
-			fmt.Fprintf(sb, "WGPU%s%s %s, ", PascalCase(o.Name), typeSuffix, CamelCase(o.Name))
+			fmt.Fprintf(sb, "%s %s, ", g.CType(o.Base, ""), CamelCase(o.Name))
 		} else {
-			fmt.Fprintf(sb, "WGPU%s%s %s", PascalCase(o.Name), typeSuffix, CamelCase(o.Name))
+			fmt.Fprintf(sb, "%s %s", g.CType(o.Base, ""), CamelCase(o.Name))
 		}
 	}
 	for i, arg := range f.Args {
 		if arg.Optional {
 			sb.WriteString("WGPU_NULLABLE ")
 		}
-		typeSuffix := g.TypeSuffixForNamespace(arg.Namespace)
 		matches := arrayTypeRegexp.FindStringSubmatch(arg.Type)
 		if len(matches) == 2 {
 			fmt.Fprintf(sb, "size_t %sCount, ", CamelCase(Singularize(arg.Name)))
-			fmt.Fprintf(sb, "%s %s", g.CType(matches[1], arg.Pointer, typeSuffix), CamelCase(arg.Name))
+			fmt.Fprintf(sb, "%s %s", g.CType(matches[1], arg.Pointer), CamelCase(arg.Name))
 		} else {
-			fmt.Fprintf(sb, "%s %s", g.CType(arg.Type, arg.Pointer, typeSuffix), CamelCase(arg.Name))
+			fmt.Fprintf(sb, "%s %s", g.CType(arg.Type, arg.Pointer), CamelCase(arg.Name))
 		}
 		if i != len(f.Args)-1 {
 			sb.WriteString(", ")
 		}
 	}
 	if f.Callback != nil {
-		fmt.Fprintf(sb, ", %s callbackInfo", g.CType(*f.Callback, "", "Info"))
+		fmt.Fprintf(sb, ", %s callbackInfo", g.CType(*f.Callback, "")+"CallbackInfo")
```

Comment:
nit: I think I would expect spaces around this `+`. Have you run gofmt? (I forget if we have a presubmit check for gofmt)

Reply:
I have autoformatting on, so surprisingly, it actually prefers the no space version. I ended up rolling the constant string into the format string for this instead.